boards/DCM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -0,0 +1,91 @@
+#include "App_SharedSetPeriodicCanSignals.h"
+#include "App_SetPeriodicCanSignals.h"
+#include "App_InRangeCheck.h"
+
+STATIC_DEFINE_APP_SET_PERIODIC_CAN_SIGNALS_IN_RANGE_CHECK(DcmCanTxInterface)
+
+// TODO: Implement PID controller to maintain DC bus power at 80kW
+// #680
+void App_SetPeriodicCanSignals_TorqueRequests(const struct DcmWorld *world)
+{
+    struct DcmCanRxInterface *can_rx = App_DcmWorld_GetCanRx(world);
+    struct DcmCanTxInterface *can_tx = App_DcmWorld_GetCanTx(world);
+
+    const float MAX_SAFE_TORQUE_REQUEST_NM =
+        21.0f; // 21Nm is max torque each motor can handle
+
+    float regen_paddle_percentage =
+        (float)App_CanRx_DIM_REGEN_PADDLE_GetSignal_MAPPED_PADDLE_POSITION(
+            can_rx);
+    float torque_request;
+
+    // Regen allowed when braking or (speed > 5kmh and AIRs closed)
+    float regen_allowed_threshold_kph = 5.0f;
```

Comment:
Actually how about we just name this `regen_threshold_kph` and then `is_vehicle_above_regen_allowed_threshold` can be renamed to `is_vehicle_over_threshold`.

Reply:
yes that is better, done